### DISCUSSION TITLE
Finetunning the models

### DISCUSSION THREAD

**Discussion by leonnn1** [JoePenna/Dreambooth-Stable-Diffusion#110] [Q&A]
I have some great models that I experimented on, but I think that they lack on clarity sometimes and I wanted to finetune them.
First question is to how.
I noticed that there are yaml files inside of the configs/stable-diffusion folder one is the finetune, finetune_unfrozen and inference, I know that the one that is used for training is finetune_unfrozen but I wonder if we could somehow finetune better for detailes with some of these yaml files or to customize our own and if so, does anyone have tested this and have something to add?
I wonder if someone tried to use the inference one and reached some good results.

**Comment by nawnie** [comment]
Would love some more info on this as well, some parameters are well notated but most are not
